app/src/test/java/com/codesoom/assignment/session/utils/JwtUtilTest.java

```diff
@@ -0,0 +1,53 @@
+package com.codesoom.assignment.session.utils;
+
+import com.codesoom.assignment.session.errors.InvalidTokenException;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.security.SignatureException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class JwtUtilTest {
+    private JwtUtil jwtUtil;
+    private static final String SECRET = "12345678901234567890123456789012";
+
+    private static final String VALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.ZZ3CUl0jxeLGvQ1Js5nG2Ty5qGTlqai5ubDMXZOdaDk";
+    private static final String INVALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.ZZ3CUl0jxeLGvQ1Js5nG2Ty5qGTlqai5ubDMXZOdaDz";
```

Comment:
무엇이 달라졌는지 파악하기가 어렵네요. 이렇게 하는 것은 어떨까요?

```suggestion
    private static final String VALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.ZZ3CUl0jxeLGvQ1Js5nG2Ty5qGTlqai5ubDMXZOdaDk";
    private static final String INVALID_TOKEN = VALID_TOKEN + "wrong";
```

Reply:
훨씬 명확한 것 같습니다!